src/utils/satelliteUtils.ts

```diff
@@ -1,17 +1,25 @@
 import { Satellite } from './../db/schema/satellite'
+import SatelliteInterface from '../utils/types/satelliteInterface'
 export default class SatelliteUtilities {
 
-    static async createSatelite(newSatelite: Record<string, string>): Promise<void> {
-        await Satellite.create(newSatelite)
-            .catch((err) => {
-                console.log(err)
-                throw 'error occured while putting new satellite'
-            })
-        return Promise.resolve()
+    static async createSatelite(newSatelite: SatelliteInterface): Promise<void> {
+        try {
+            await Satellite.create(newSatelite)
```

Comment:
as much as I prefer async/await, this project is about you learning promises, stick with them.